src/plugins/drawing/model/drawing-content.test.ts

```diff
@@ -625,29 +616,6 @@ describe("DrawingContentModel", () => {
     expect(model.currentStamp!.url).toBe("b.png");
   });
 
-  it("can update image urls", () => {
-    const originalUrl = "my/image/url";
-    const image = ImageObject.create({
-      url: originalUrl, x: 0, y: 0, width: 10, height: 10
-    });
-    const model = createDrawingContentWithMetadata({
-      objects: [image]
-    });
-
-    model.updateImageUrl("", "");
-    expect(image.url).toEqual(originalUrl);
-
-    // Updates to a empty string are ignored
-    model.updateImageUrl("my/image/url", "");
-    expect(image.url).toEqual(originalUrl);
-
-    model.updateImageUrl("", "my/image/newUrl");
-    expect(image.url).toEqual(originalUrl);
-
-    model.updateImageUrl("my/image/url", "my/image/newUrl");
-    expect(image.url).toBe("my/image/newUrl");
-  });
-
```

Comment:
I think this updateImageUrl action is still used. I think it is called in `ImageToolComponent`.